.github/workflows/build-and-push-to-docker.yml

```diff
@@ -182,27 +151,28 @@ jobs:
         username: ${{ secrets.RENKU_DOCKER_USERNAME }}
         password: ${{ secrets.RENKU_DOCKER_PASSWORD }}
 
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Build renku project python-based docker image extensions
       id: build-image
       run: |
-        cd docker/cuda
+        # cd docker/cuda
 
         if [[ ${{ github.ref }} == refs/tags* ]]; then
           export LABEL=$(echo ${{ github.ref }} | cut -d / -f 3)
         else
           export LABEL=$(echo ${{ github.sha }} | cut -c 1-7)
         fi
 
-        docker build . \
-          --build-arg BASE_IMAGE="renku/renkulab-py:python-${{ matrix.PYTHON_VERSION }}-$LABEL" \
-          --build-arg CUDA_VERSION="${{ matrix.CUDA_VERSION }}" \
-          --build-arg EXTRA_LIBRARIES="${{ matrix.EXTRA_LIBRARIES }}" \
-          --build-arg CUDA_CUDART_PACKAGE="${{ matrix.CUDA_CUDART_PACKAGE }}" \
-          --build-arg CUDA_COMPAT_PACKAGE="${{ matrix.CUDA_COMPAT_PACKAGE }}" \
-          --build-arg LIBCUDNN_PACKAGE="${{ matrix.LIBCUDNN_PACKAGE }}" \
-          --tag $DOCKER_NAME-cuda:${{ matrix.CUDA_VERSION }}-$LABEL
-        echo "IMAGE_NAME=$DOCKER_NAME-cuda:${{ matrix.CUDA_VERSION }}-$LABEL" >> $GITHUB_OUTPUT
+        export DOCKER_PREFIX="renku/renkulab"    
+        export DOCKER_LABEL="${{ matrix.CUDA_VERSION }}-${LABEL}"
+        export RENKU_BASE="renku/renkulab-py:python-${{ matrix.PYTHON_VERSION }}-$LABEL" 
+        export CUDA_VERSION="${{ matrix.CUDA_VERSION }}" 
+        export EXTRA_LIBRARIES="${{ matrix.EXTRA_LIBRARIES }}" 
+        export CUDA_CUDART_PACKAGE="${{ matrix.CUDA_CUDART_PACKAGE }}" 
+        export CUDA_COMPAT_PACKAGE="${{ matrix.CUDA_COMPAT_PACKAGE }}" 
+        export LIBCUDNN_PACKAGE="${{ matrix.LIBCUDNN_PACKAGE }}" 
+        make cuda
+        echo "IMAGE_NAME=$DOCKER_NAME-cuda:$DOCKER_LABEL" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
        echo "IMAGE_NAME=$DOCKER_PREFIX-cuda:$DOCKER_LABEL" >> $GITHUB_OUTPUT
```

Reply:
Oh I see what's going on - `DOCKER_NAME` is defined above but `DOCKER_PREFIX` is used in the `Makefile` - why not just one or the other?